Externals.cfg

```diff
@@ -18,19 +18,19 @@ required = True
 
 [ufs-weather-model]
 protocol = git
-repo_url = https://github.com/ufs-community/ufs-weather-model
+repo_url = https://github.com/NOAA-EPIC/ufs-weather-model-1
 # Specify either a branch name or a hash but not both.
```

Comment:
@mark-a-potts, should this point to ufs-community and not NOAA-EPIC?  Or is this just for testing?

Reply:
OK, we'll just have to see if the WoFS SDF XML will work without the pointer updates.  I've emailed @ligiabernardet  and @grantfirl to ask about the status of the v6 CCPP release branch and whether it includes the major module updates that went into the ufs-weather-model repo recently.

Reply:
Thanks, @JeffBeck-NOAA  I emailed Grant as well. He had commits to release-v6 today, so I am thinking that even if they are ready on their side, the changes at the fv3atm level have not been merged yet. @arunchawla-NOAA recommended on slack that we should probably delay the release of the SRW. I am on the fence, but if we do the release, I don't think it can be with ccpp-v6.

Reply:
@mark-a-potts, just read through the email threads related to the physics/fv3atm release branches.  I would agree with your assessment that it's best to delay by about a week.  I'd hope it won't take too long to get the branch situation straightened out, but there are several submodule branches/pointers that have to be lined up with the correct cherry-picked PRs.  It looks like Grant has taken a big step already toward sorting this out.  However, I'm wondering if the resulting release branch tree will still require the major module updates that currently don't work in the App.  If that does turn out to be the case, I'm happy to quickly track down the required changes to srw_common and get a PR open into the App-level release branch.

Reply:
I agree. Let's postpone the release and focus on getting the physics/branch/submodule situation lined up to everyone's satisfaction. I think we add considerable value to the release by incorporating the latest physics package.